### PR TITLE
change prompt usvc prot to 6018

### PR DESCRIPTION
**Pull request by shaohef**
## Description

6012, 6013, 6014 is used by chathistory.
6016 is used by feedback.
To avoid unnecessary confusion and conflict, change it to 6018.

## Issues

`n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

`n/a`.

## Tests

No special test steps, just follow the README

### CODE REVIEW

File: comps/prompt_registry/mongo/README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The Prompt Registry microservice exposes the following API endpoints:
 
   ```bash
   curl -X 'POST' \
-    http://{host_ip}:6012/v1/prompt/create \
```
Comment (yongfengdu) [inline comps/prompt_registry/mongo/README.md:53]: s/{host_ip}/${host_ip}/
The same for all {host_ip} lines to reference an environmental var.

**Comment by hteeyeoh** [comment]
Looks good to me. Approved

**Review by hteeyeoh — APPROVED** [review]

**Review by lvliang-intel — APPROVED** [review]

**Review by yongfengdu — APPROVED** [review]

**Review by hteeyeoh — APPROVED** [review]

**Comment by yongfengdu** [comment]
Suggest to update the docker images (https://hub.docker.com/r/opea/promptregistry-mongo-server/tags) after this PR merged. Otherwise, the docker compose/README(Use port 6018) will be inconsistent with the image(Use port 6012).

**Comment by chensuyue** [comment]
> @chensuyue Suggest to update the docker images (https://hub.docker.com/r/opea/promptregistry-mongo-server/tags) after this PR merged. Otherwise, the docker compose/README(Use port 6018) will be inconsistent with the image(Use port 6012).

The release image based on tag v1.0. Should you update GenAIExamples as well?

**Comment by hteeyeoh** [comment]
I will take care of to update in ProductivitySuite example which use this image  in GenAIExample

**Pull request merged**